pyrit/prompt_converter/rot13_converter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import codecs
+
+from pyrit.prompt_converter import PromptConverter
+
+
+class ROT13Converter(PromptConverter):
+    def convert(self, prompt: str) -> str:
```

Comment:
```suggestion
    def convert(self, prompt: str) -> list[str]::
```
Updating based on the PR that was pushed to main in the meantime.